### PR TITLE
[FIX] Missing Privacy Policy Agree on register

### PR DESCRIPTION
**Pull request by geekgonecrazy**
image
Moving here ensures that its agreed to if they chose to register.

### CODE REVIEW

**Comment by reetp** [comment]
For GDPR I think you will find you cannot pre check Marketing/Email check boxes..... it must be voluntary Opt In. It should probably also have  a positive consent reply as well.

I also think you should have a think about the Terms and Privacy policies depending on if Rocket is self hosted or cloud based as there may be considerable differences in the transmitted and how it is handled. One size is not fitting all and there is no real clarity between self hosted and cloud.

**Comment by reetp** [comment]
https://github.com/RocketChat/feature-requests/issues/135

**Comment by geekgonecrazy** [comment]
@reetp that part hasn't changed in the PR.  But I did wonder about that.  Let me confirm and I can get that changed as an additional change.

File: client/components/setupWizard/steps/RegisterServerStep.js [hunk 1]
```diff
@@ -153,6 +163,19 @@ export function RegisterServerStep({ step, title }) {
 						<Item icon='circle'>{t('Register_Server_Standalone_Own_Certificates')}</Item>
 					</Items>
 				</Option>
+
+				<Label text={<>{t('Register_Server_Registered_I_Agree')} <a href='https://rocket.chat/terms'>{t('Terms')}</a> & <a href='https://rocket.chat/privacy'>{t('Privacy_Policy')}</a></>} position='end' className='SetupWizard__RegisterServerStep__PrivacyTerms'>
```
Comment (tassoevan) [inline client/components/setupWizard/steps/RegisterServerStep.js]: @ggazzo Friendly reminder that we need to find a proper way to interpolate translations!

Comment (geekgonecrazy) [reply client/components/setupWizard/steps/RegisterServerStep.js]: This felt pretty terrible doing it.  But the output of t was always a string when I tried to use the typical %s variable substitution.  Which I guess makes sense

**Pull request merged**